packages/catalog-model/examples/spotify-api.yaml

```diff
@@ -6,9 +6,9 @@ metadata:
   tags:
     - spotify
     - rest
-  annotations:
-    backstage.io/definition-at-location: 'url:https://raw.githubusercontent.com/APIs-guru/openapi-directory/master/APIs/spotify.com/v1/swagger.yaml'
 spec:
   type: openapi
   lifecycle: production
   owner: spotify@example.com
+  definition:
```

Comment:
Since this is an example that's loaded into peoples local deployments we should make it backwards compatible. Could you keep the old annotation with a comment about deprecation and removal in a future update?

Reply:
Ah, you mean that the GitHub master state stays in a way that every existing instance keeps working? Yeah, that's a good idea.